woudc_extcsv/__init__.py

```diff
@@ -115,8 +115,8 @@ def __init__(self, content, parse_tables=False, encoding='utf-8'):
                 body.replace('%', ',')
             try:
                 s = StringIO(body)
-                c = csv.reader(s, encoding=encoding)
-            except Exception as err:
+                c = csv.reader(s)
+            except Exception:
```

Comment:
Can you add back in and log the error?
```python
except Exception as err:
    LOGGER.warning(err)
    c = csv.reader(s)
```

Reply:
Do you want this to replace the existing line? Most of the other try-catches use this method of logging, should they all be changed to use the logger?

`self.errors.append(_violation_lookup(0))`